client/src/research/DynamicTable.js

```diff
@@ -344,12 +230,105 @@ export default class DynamicHeightTableColumn extends React.PureComponent {
           dataKey="responseText"
           label="Response"
           disableSort = {false}
-          cellDataGetter={({ dataKey , rowData }) => rowData.json[dataKey] || (this._getAudioUrl(rowData) && <div>{this._loadAudioPlayer(this._getAudioID(this._getAudioUrl(rowData)))} {this._loadTranscript(this._getAudioID(this._getAudioUrl(rowData)))} </div>)}
-          cellRenderer= {this._wrappingCellRendererAgain}
+          cellDataGetter={({ dataKey , rowData }) => rowData.json[dataKey] || (this._getAudioUrl(rowData) && <div><AudioPlayerComponent audioID={this._getAudioID(this._getAudioUrl(rowData))} token={this.props.token} /> <TranscriptComponent audioID={this._getAudioID(this._getAudioUrl(rowData))} token={this.props.token}/> </div>)}
+          cellRenderer= {this._wrappingCellRenderer}
           headerRenderer={this._headerRenderer}
           width={width}
         />
       </Table>
     );
   }
-}
\ No newline at end of file
+}
+
+class AudioPlayerComponent extends React.Component {
```

Comment:
This looks great.  Can you move `AudioPlayerComponent` and `TranscriptComponent` into their own files?  They're now nicely self-contained which simplifies the table code quite a bit but also could be re-used if other folks are making other kinds of analysis.  In the same file here they're a bit hidden away.

Reply:
done